src/ceph-disk/ceph_disk/main.py

```diff
@@ -777,8 +803,8 @@ def get_partition_base(dev):
     Get the base device for a partition
     """
     dev = os.path.realpath(dev)
-    if not stat.S_ISBLK(os.lstat(dev).st_mode):
```

Comment:
the lstat is useless here: os.path.realpath will return the actual file, not the symlink, therefore dev will never be a symlink